src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,252 @@
+                }
+                break;
+            case "delete":
+                try {
+                    int taskIndex = Integer.parseInt(lineParts[1]) - 1;
+                    deleteTask(taskIndex);
+                } catch (NumberFormatException e) {
+                    System.out.print(lineParts[1] + " is not a valid number.\n");
+                } catch (DukeException e) {
+                    System.out.print("That is not a valid task index, please try again.\n");
+                }
+                break;
+            default:
+                mockEcho(line);
+                System.out.print("Use -h for list or available commands.\n");
+            }
+            printSeparator();
+            line = in.nextLine();
+        }
+        exitMethod();
+    }
+
+    private static void loadFile(Path path) {
+        //java.nio.file.Files.exists(path)
+        try {
+            File f = new File(path.toString());
+            if (f.createNewFile()) {
+                System.out.println("File created: " + path.toAbsolutePath());
+            } else {
+                System.out.println("File already exists.\nLoading...");
+                Scanner s = new Scanner(f); // create a Scanner using the File as the source
+                while (s.hasNext()) {
+                    String[] lineParts = s.nextLine().split(FILE_SEPARATOR);
+                    switch(lineParts[0]) {
+                    case "Todo":
+                        try {
+                            addTask(new Todo(lineParts[2], Boolean.parseBoolean(lineParts[1])));
+                        } catch (DukeException e) {
+                            System.out.print("The description of a todo cannot be empty.\n");
+                        }
+                        break;
+                    case "Deadline":
+                        try {
+                            addTask(new Deadline(lineParts[2], lineParts[3], Boolean.parseBoolean(lineParts[1])));
+                        } catch (IndexOutOfBoundsException e) {
+                            System.out.print("Something went wrong. Please put the due date after /by.\n");
+                        } catch (DukeException e) {
+                            System.out.print("The description of a deadline cannot be empty.\n");
+                        }
+                        break;
+                    case "Event":
+                        try {
+                            addTask(new Event(lineParts[2], lineParts[3], Boolean.parseBoolean(lineParts[1])));
+                        } catch (IndexOutOfBoundsException e) {
+                            System.out.print("Something went wrong. Please put the event time after /at.\n");
+                        } catch (DukeException e) {
+                            System.out.print("The description of a event cannot be empty.\n");
+                        }
+                        break;
+                    }
+                }
+            }
```

Comment:
There are a couple of indentations here, which could obstruct reading flow. See [https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent) for suggested improvements.

Reply:
Switched it around to make the happy path prominent. Thank you for the advice